tests/unit/maze_transformer/training/test_dataset.py

```diff
@@ -0,0 +1,35 @@
+import torch
```

Comment:
Did you mean to include this file or was it part of your other PR?

Reply:
Nope, cheers!